.circleci/config.yml

```diff
@@ -0,0 +1,188 @@
+version: 2
+jobs:
+    python-3.5: &build
+        docker:
+            - image: circleci/jdk8:0.1.1
+
+        steps:
+            - checkout
+            - restore_cache:
+                key: coala-bears-dep-{{checksum "requirements.txt"}}
+
+            - run:
+                name: dependencies
+                command: |
+                  sudo apt-get update
+                  sudo apt-get install -y software-properties-common python-software-properties curl
+                  sudo add-apt-repository -y ppa:hvr/ghc
+                  sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+                  sudo add-apt-repository -y ppa:avsm/ppa
+                  curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash  -
+                  curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+                  echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+                  sudo apt-get update
+                  sudo apt-get install -y python3.4-venv git bzip2 \
+                  cabal-install-1.24 clang-3.4 libreadline-dev \
+                  gfortran ghc happy indent libblas-dev liblapack-dev \
+                  libperl-critic-perl libxml2-utils menhir php-codesniffer \
+                  build-essential ruby texinfo libbz2-dev libcurl4-openssl-dev \
+                  libexpat-dev libncurses-dev zlib1g-dev git-core libssl-dev \
+                  libsqlite3-dev sqlite3 libxslt1-dev libffi-dev yarn python3-dev \
+                  libgdbm-dev libncurses5-dev automake libtool bison \
+                  libncursesw5-dev libc6-dev tk-dev
+
+            - run:
+                name: virtual-env
+                command: |
+                  if [ ! -d "$HOME/.venv" ]; then
+                  python3 -m venv ~/.venv
```

Comment:
why do you need a venv here, if you are using `pyenv` below?

Reply:
I remember you had told to remove virtual-env from circleci 2.0 from the coala/coala code base
Wanted to remove this and cache it but some had some problems with it...
Then I thought that I should first get a green build and then try for this

Or can it create a conflict with some packages??